wait/http_test.go

```diff
@@ -247,3 +250,212 @@ func TestHTTPStrategyWaitUntilReadyNoBasicAuth(t *testing.T) {
 		return
 	}
 }
+
+func TestHttpStrategyFailsWhileGettingPortDueToOOMKilledContainer(t *testing.T) {
+	var mappedPortCount int
+	target := &wait.MockStrategyTarget{
+		HostImpl: func(_ context.Context) (string, error) {
+			return "localhost", nil
+		},
+		MappedPortImpl: func(_ context.Context, _ nat.Port) (nat.Port, error) {
+			defer func() { mappedPortCount++ }()
+			if mappedPortCount == 0 {
+				return "", errors.New("port not found")
```

Comment:
We could probably extract `errors.New("port not found")` to a const/var to avoid duplications. It could be also interesting adding it to http.go/host_port.go/sql.go as an exported var to be reused as a type. wdyt?

Reply:
This error mimics the error the code below (in L171).

https://github.com/testcontainers/testcontainers-go/blob/447099b908d12a0dd64f4110e995c51ea2933751/docker.go#L144-L172

Perhaps the const/var should be defined in the `testcontainers-go` package, but referencing it from `wait` package is a cycle import.

Since this is just a error for testing, I don't think it makes much sense to avoid duplication.

Reply:
Thank you for comment.
I extracted the error into var.
How is this?